zkevm-circuits/src/keccak_circuit/table.rs

```diff
@@ -113,3 +127,168 @@ pub(crate) fn load_lookup_table<F: Field>(
         },
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use halo2_proofs::circuit::SimpleFloorPlanner;
+    use halo2_proofs::dev::{CellValue, MockProver};
+    use halo2_proofs::halo2curves::bn256::Fr as F;
+    use halo2_proofs::plonk::{Circuit, ConstraintSystem};
+    use itertools::Itertools;
+    use std::iter::zip;
+
+    #[test]
+    fn normalize_table() {
+        normalize_table_impl(3, 10);
+        normalize_table_impl(4, 10);
+        normalize_table_impl(6, 10);
+        normalize_table_impl(6, 19);
+    }
+
+    fn normalize_table_impl(range: usize, log_height: usize) {
+        let table = build_table(&TableTestCircuit {
+            range,
+            log_height,
+            normalize_else_chi: true,
+        });
+
+        // On all rows, all inputs/outputs are correct, i.e. they have the same low bit.
+        assert_eq!(BIT_COUNT, 3);
```

Comment:
What's the point of asserting the constant? Do we want to hightlight the table depends on `BITCOUNT = 3`?
If that's the case we can add a comment for it.

Reply:
Done.